include/diy/collection.hpp

```diff
@@ -23,16 +23,16 @@ namespace diy
       typedef       detail::Load                                Load;
 
     public:
-                    Collection(Create               create,
-                               Destroy              destroy,
-                               ExternalStorage*     storage,
-                               Save                 save,
-                               Load                 load):
-                        create_(create),
-                        destroy_(destroy),
-                        storage_(storage),
-                        save_(save),
-                        load_(load),
+                    Collection(Create               p_create,
```

Comment:
Why all these changes?

Reply:
The parameters shadow methods on the class. The other renames are for similar reasons. If you have other names that you'd prefer, I can change them.

Reply:
Oh, I see. Which compiler complains about this?

Reply:
GCC does. You can see the warnings [here](https://open.cdash.org/viewBuildError.php?type=1&buildid=5242544).

Reply:
I see. I can't quite tell what version of GCC that is, but I guess it doesn't really matter. Let me think about this a little. I usually prefer to deal with such problems by adding underscores, but that won't work here, since both `create()` and `create_` are in use. I guess you could call the parameters `create__`, but that's also on the ugly side.

Reply:
Yeah, I wasn't sure what was best either :) . I don't care too much about naming, but fixing the warnings in some way would be nice.